docs/references/creating-a-release.md

```diff
@@ -10,41 +10,75 @@ We must do due diligence in testing our packages before any new major/minor or b
+Both beta and stable release follow a two step workflow, aided by github actions.
 
-While the release branch is active, some changes might be merged into the branch (for bug fixes, features deemed necessary for the release). PRs into the release branch should follow this process when possible:
+### Step 1: Prepare for release
 
-- First land the change as a PR into `main`.
-- Then, cherry-pick the change as a separate PR onto the release branch.
+Use the <insert workflow name> github action to trigger the release preparation workflow.
 
-This process has the following benefits:
+    TODO: Add image showing workflow and options, like there used to be here.
 
-- The release branch never diverges off of `main`. In theory, it is possible to abandon the release branch at any point and create a new one off of `main` without losing work.
--  All PR reviews happen on `main`, and the cherry-pick PR simply requires a sign-off. This avoids non-trivial merge conflicts when the release branch is eventually merged back into `main`.
+This workflow will:
+
+1. Create a `prerelease/<release-tag>` branch, bump the package version for `@azure/communication-react` as appropriate and collect all change files into a changelog.
+1. Create a Pull Request into the `prerelease/<release-tag>` for manually summarizing the changes collected in the changelog. *Merge this PR before going to step 2*.
+
+For example, when creating a release off of `main` tagged `3.3.0`, the following branches and Pull Requests will be created:
+
+```mermaid
+graph LR
+  main[branch: main]
+  prerelease[branch: prerelease/3.3.0<br/>- bump version to 3.3.0<br/>- collect changelog<br/>-trigger string translations]
+  pr[branch:groom_changelog_3.0.0]
+
+  main -->|Create Branch| prerelease
+  prerelease -->|Create Branch| pr
+  pr -.-o|Create Pull Request| prerelease
+```
+
+### Step 2: Create release branch
+
+Use the <insert workflow name> github action to trigger the release branch creation workflow.
+
+    TODO: Add image showing workflow and options, like there used to be here.
 
-When ready to release to NPM, a new package is created off the release branch and the release branch is then merged into `main` and deleted. A git tag is added to the released git-version.
+This workflow will:
 
-## Creating a release through GitHub actions (Preferred)
+1. Create a Pull Request to merge the prerelease branch back into the base branch.
+  1. For stable release only: Before creating the Pull Request, @azure/communiation-react will be version bumped once again to a `-beta.0` version (this ensures that we can continue to add changes of `prerelease`) type.
+1. Create a new release branch off of the prerelease branch. This branch will be used for the eventual release, but *it will never be merged back in the base branch*.
+  1. Commit some release preparation modifications that can not be merged back, like choosing the dependency versions that depend on the type of the release.
```

Comment:
TODO: List all the steps here.

TODO: Include https://github.com/Azure/communication-ui-library/pull/1642

Reply:
TODO: sync telemetry version and ensure versions match